### ISSUE TITLE
Network interfaces for instance don't need to be paginated

### ISSUE THREAD

**Issue by david-crespo** [oxidecomputer/omicron#1009]
I believe there's a maximum of 8 or 6 or some small number, so there's no need to paginate.

https://github.com/oxidecomputer/omicron/blob/810301c3aa15cf297a95b6880824f21c0ee9aca1/nexus/src/external_api/http_entrypoints.rs#L1475-L1488

**Comment by rmustacc** [comment]
Correct, right now there is a max of 8 per instance (related to PCI hotplug).

**Comment by david-crespo** [comment]
It's fine as-is.

**Comment by karencfv** [comment]
It'd be really nice if they weren't paginated. That way they could be included as part of the `Instance` object. Is it OK to reopen this issue?
